python/setup.py

```diff
@@ -31,7 +31,7 @@
 docs = ["sphinx"]
 video = ["ffmpeg-python", "scenedetect"]
 youtube = ["pafy", "youtube_dl"]
-mlflow = ["mlflow>=1.15"]
+mlflow = ["mlflow==1.20.2"]
```

Comment:
is it safe to pin it to the minor version at least?

Reply:
not safe, as mentioned in the pull request body

Reply:
From the docs https://mlflow.org/docs/latest/rest-api.html#get-latest-modelversions
and my local curl tests. The mlflow server doesn't obey the docs (docs say it should be POST, but it's actually GET), so we must choose the client version that also doesn't obey the docs. We can't tell if a client version works as we expected before a test.
```
(rikai) ➜  python git:(renka-set-model-tag) ✗ curl -X GET http://localhost:5000/api/2.0/mlflow/registered-models/get-latest-versions -d '{"name":"jb", "stages": []}'
{"error_code": "RESOURCE_DOES_NOT_EXIST", "message": "Registered Model with name=jb not found"}%
(rikai) ➜  python git:(renka-set-model-tag) ✗ curl -X POST http://localhost:5000/api/2.0/mlflow/registered-models/get-latest-versions -d '{"name":"jb", "stages": []}'
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>405 Method Not Allowed</title>
<h1>Method Not Allowed</h1>
<p>The method is not allowed for the requested URL.</p>
```

Reply:
lol mlflow ftl